connectors/sources/generic_database.py

```diff
@@ -103,6 +105,16 @@ def get_default_configuration(cls):
                 "label": "Oracle connection protocol",
                 "type": "str",
             },
+            "oracle_home": {
```

Comment:
why are the oracle options in the generic database class? shouldn't they be in the oracle class?

Reply:
Sure, moved in `oracle.py` file. Also applied a similar change in PostgreSQL